account_mass_reconcile_transaction_ref/__manifest__.py

```diff
@@ -1,9 +1,8 @@
-# -*- coding: utf-8 -*-
 # © 2013-2018 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
 {
     'name': 'Mass Reconcile Transaction Ref',
-    'version': '10.0.1.1.0',
+    'version': '11.0.1.1.0',
```

Comment:
not sure what's the policy for this repo or module but this should be `11.0.1.0.0`.
Personally, I'm not against keeping the same version tho ;)